devfiles/quarkus-kotlin/devfile.yaml

```diff
@@ -0,0 +1,91 @@
+---
+apiVersion: 1.0.0
+metadata:
+  generateName: kotlin-
+projects:
+  - name: rest-kotlin-quickstart
+    source:
+      location: 'https://github.com/blues-man/rest-kotlin-quickstart.git'
+      type: git
+      branch: master
+components:
+  - id: blues-man/quarkus-kotlin/latest
```

Comment:
This will change once the corresponding PR in the plugin registry is updated. Otherwise the rest of the file LGTM.